backend/server/controllers/assistants.js

```diff
@@ -127,7 +127,7 @@ module.exports = {
         })
         const isTeacher = await helper.getTeacherId(req.decoded.id, courseInstance.id)
         if (!isTeacher) {
-          return res.status(403).send('must be teacher on the course')
+          return res.status(403).send('must be instructor on the course')
```

Comment:
I'll approve, but the error messages could be modified to be with an uppercase initial letter and end with a period (etc) while you're modifying them.